### PR TITLE
Feature/Progress bar seek

### PR DESCRIPTION
**Pull request by ValeriiMalkov**
Progress bar was added to media application view.
Also there is was added an `OnSeekMediaClockTimer` notification to SDL

Need tests

### CODE REVIEW

File: app/model/sdl/MediaModel.js [hunk 1]
```diff
@@ -219,6 +249,31 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
 
   }.observes('this.currTime'),
 
+  seekTracking: function(event){
+    if(event.which == 1){
+      this.set('pause', true);
+      var progressBar = document.querySelector('progress');
+      var percent = event.offsetX / progressBar.offsetWidth;
+      progressBar.value = percent * 100;
+      if(this.countUp){
```
Comment (BSolonenko) [inline app/model/sdl/MediaModel.js]: please use a ternary operator

File: app/model/sdl/MediaModel.js [hunk 2]
```diff
@@ -259,10 +314,23 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
...
+          }
+        }
+        else {
+          this.set('seekBar', false);
+        }
       }
       this.set('pause', false);
     }
-
+
```
Comment (BSolonenko) [inline app/model/sdl/MediaModel.js]: please remove white space

File: app/model/sdl/MediaModel.js [hunk 3]
```diff
@@ -188,15 +195,38 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
...
+        return;
+      }
     } else {
       if (this.duration <= this.currTime) {
         clearInterval(this.timer);
         this.currTime = 0;
         this.appInfo.set('mediaClock', '00:00:00');
         return;
       }
+      if (this.endTime != 0 && (this.duration - this.endTime) <= this.currTime){
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/MediaModel.js]: @ValeriiMalkov please merge this if with previous one as they have the same body

File: app/model/sdl/MediaModel.js [hunk 4]
```diff
@@ -178,6 +179,12 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
     }
   }.observes('this.pause'),
 
+  deleteProgressBar: function() {
```
Comment (AKalinich-Luxoft) [inline app/model/sdl/MediaModel.js]: @ValeriiMalkov 
1. Please provide a description for this function
2. Suggest to rename it to `hideProgressBar`
3. Please check for a test case when progress bar was hidden -> show -> hide -> show. I guess this function removes it from view permanently

**Pull request merged**